lib/node_modules/@stdlib/blas/ext/base/dnannsumpw/README.md

```diff
@@ -165,8 +165,132 @@ console.log( out );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
 * * *
 
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/blas/ext/base/dnannsumpw.h"
+```
+
+#### stdlib_strided_dnannsumpw( N, \*X, strideX, \*n )
+
+Computes the sum of double-precision floating-point strided array elements, ignoring `NaN` values and using pairwise summation.
+
+```c
+const double x[] = { 1.0, 2.0, 0.0/0.0, 4.0 };
```

Comment:
```suggestion
#include "stdlib/blas/base/shared.h"

const double x[] = { 1.0, 2.0, 0.0/0.0, 4.0 };
```

We need to include the shared header; otherwise, `CBLAS_INT` is not defined.